src/test/resources/application-test.yaml

```diff
@@ -22,6 +22,8 @@ form-flow:
       domain: 'mail.forms-starter.cfa-platforms.org'
       sender-email: 'Testing <test@mail.forms-starter.cfa-platforms.org>'
 spring:
+  main:
+    allow-bean-definition-overriding: true
```

Comment:
This allows me to inject the mocked address validation service. A good pattern for things like that, apparently.